src/SUMMARY.md

```diff
@@ -1,6 +1,8 @@
 # Rust Forge
 
 [Overview](./README.md)
+- [Core](./core/README.md)
+    - [The rust blogs](./core/blogs.md)
```

Comment:
This should probably be "Rust Blog guidelines" to match the bit below

arguably "guidelines" should be capitalized in both places as well